ios-target/View/CustomControls/UIStoryboardSegueFromLeftToRight.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  UIStoryboardSegueFromLeftToRight.swift
+//  ios-target
+//
+//  Created by sol manini on 7/10/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class UIStoryboardSegueFromLeftToRight: UIStoryboardSegue {
+  
+  override func perform() {
+    let sourceController = source as UIViewController
+    let destinationController = destination as UIViewController
+  
+    sourceController.view.superview?.insertSubview(destinationController.view,
+                                                   aboveSubview: sourceController.view)
+    destinationController.view.transform = CGAffineTransform(translationX: -sourceController.view.frame.size.width, y: 0)
+    
+    UIView.animate(withDuration: 0.35,
```

Comment:
Also, if you want, you can try playing a bit with [animationOptions](https://developer.apple.com/documentation/uikit/uiview/animationoptions) like `curveEaseInOut` to get your transition to look smoother 😁.